fairlearn/widget/fairlearn_dashboard.py

```diff
@@ -22,11 +22,9 @@ class FairlearnDashboard(object):
     """The dashboard class, wraps the dashboard component.
 
     :param sensitive_features:  A matrix of feature vector examples (# examples x # features),
-        these can be from the initial dataset, or reserved from training. Currently only
-        categorical features are supported
+        these can be from the initial dataset, or reserved from training.
     :type sensitive_features: numpy.array or list[][] or Pandas Dataframe
-    :param y_true: The true labels for the provided dataset. Will overwrite any set on
```

Comment:
1. replace "Pandas Dataframe" by "pandas.DataFrame"
2. include explanation of is_classifier
3. do we have any support of pandas.Series? if so, we should include that here (just double checking)